package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@creativecommons/cc-assets",
+  "version": "0.1.0",
+  "description": "Assets used across Creative Commons projects.",
+  "main": "",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/creativecommons/cc-assets.git"
+  },
+  "keywords": [
+    "PNG",
+    "SVG"
+  ],
+  "author": "Creative Commons",
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
From the best I can understand, NPM has some [restrictions about what value we can use for the license property in package.json](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#license)

- an SPDX identifier - used for open-source projects
- UNLICENSED - used for private or unpublished packages
- "SEE LICENSE IN \<file_name\>" - used when the other options are not relevant

I used the third option, since it allows us to define our own license terms.